### PR TITLE
[release] Prep for v0.8.29

### PR DESCRIPTION
**Pull request by s3krit**
- [X] Bump version
- [X] Bump substrate to latest master
- [x] benchmarks

### CODE REVIEW

**Comment by s3krit** [comment]
@shawntabrizi Could you take a look at this failure? The benchbot benchmark for frame_system removed the `suicide()` weight (since the call has been removed?), leading to build failures such as https://gitlab.parity.io/parity/polkadot/-/jobs/823604

**Comment by s3krit** [comment]
Should be fixed with https://github.com/paritytech/substrate/pull/8184 and https://github.com/paritytech/polkadot/pull/2506

**Comment by s3krit** [comment]
Should be good to go - nothing crazy, just bumping version & benchmarks

File: runtime/kusama/src/weights/pallet_bounties.rs [hunk 1]
```diff
@@ -1,105 +1,104 @@
...
 	}
 	fn extend_bounty_expiry() -> Weight {
-		(36_419_000 as Weight)
+		(25_063_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn spend_funds(b: u32, ) -> Weight {
-		(7_562_000 as Weight)
+		(0 as Weight)
```
Comment (kianenigma) [inline runtime/kusama/src/weights/pallet_bounties.rs:99]: hmm fishy weight change, but I guess it is fine still.

File: runtime/kusama/src/weights/pallet_staking.rs [hunk 2]
```diff
@@ -201,14 +201,14 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
...
-			// Standard Error: 43_000
-			.saturating_add((1_426_000 as Weight).saturating_mul(v as Weight))
-			// Standard Error: 17_000
-			.saturating_add((588_000 as Weight).saturating_mul(n as Weight))
-			// Standard Error: 43_000
-			.saturating_add((69_239_000 as Weight).saturating_mul(a as Weight))
-			// Standard Error: 90_000
-			.saturating_add((8_062_000 as Weight).saturating_mul(w as Weight))
+			// Standard Error: 46_000
+			.saturating_add((712_000 as Weight).saturating_mul(v as Weight))
```
Comment (kianenigma) [inline runtime/kusama/src/weights/pallet_staking.rs:205]: hmm again this weight is almost halved. I think this means that the weight calculation was not accurate enough, not an issue with this PR.

File: runtime/polkadot/src/weights/pallet_bounties.rs [hunk 3]
```diff
@@ -42,65 +40,65 @@
...
-		(36_419_000 as Weight)
+		(25_014_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn spend_funds(b: u32, ) -> Weight {
-		(7_562_000 as Weight)
-			// Standard Error: 16_000
-			.saturating_add((77_328_000 as Weight).saturating_mul(b as Weight))
+		(0 as Weight)
```
Comment (kianenigma) [inline runtime/polkadot/src/weights/pallet_bounties.rs:99]: okay I guess this `spend_funds` one must have changed.

**Review by kianenigma — COMMENTED** [review]
The multi-phase pallet that I merged yesterday is not here 🤔 can we merge polkadot master in again?

**Review by kianenigma — APPROVED** [review]
Given that we have new pallets, we should re-run the benchmarks maybe one more time again?

**Comment by kianenigma** [comment]
(just for the new pallet)

**Comment by s3krit** [comment]
!benchmark runtime polkadot pallet_election_provider_multi_phase

**Comment by s3krit** [comment]
/benchmark runtime polkadot pallet_election_provider_multi_phase

**Comment by s3krit** [comment]
/benchmark runtime kusama pallet_election_provider_multi_phase

**Comment by s3krit** [comment]
/benchmark runtime westend pallet_election_provider_multi_phase

**Comment by s3krit** [comment]
bot merge

**Pull request merged**